services/self-service/src/main/java/com/epam/dlab/backendapi/dao/EndpointDAO.java

```diff
@@ -23,14 +23,28 @@
 
 import java.util.List;
 import java.util.Optional;
+import java.util.regex.Pattern;
 
+/**
+ * Interface of Endpoint entities.
+ */
```

Comment:
I wonder whether this comment is informative here. How do you think?

Reply:
I have updated the interface description